tests/unit/protocols/input/rest-xml.json

```diff
@@ -73,16 +73,16 @@
       {
         "given": {
           "http": {
-            "method": "PUT",
+            "method": "GET",
```

Comment:
Why change this instead of adding a new test?  If the PUT case no longer valid if you send an empty body?